libraries/protocol/include/golos/protocol/exceptions.hpp

```diff
@@ -330,6 +330,15 @@ namespace golos {
             account_already_scheduled_for_work,
             cannot_specify_owner_key_unless_creating_account,
             witness_must_be_created_before_minning,
+
+            // custom operations
+            inner_authorities_does_not_match_outer,
+
+            // database logic
+            account_exceeded_bandwidth_per_vestring_share,
+
+            // protocol logic
+            cannot_mix_posting_and_active_owner_autority_operation,
```

Comment:
We have `tx_with_both_posting_active_ops` with almost same meaning, but used in proposals. Operation types checked early in `create_proposal_operation` to prevent tx with different operation types, which (tx) is result of proposal applying.
Maybe it's better to use same logic exception for this. Or at least rename them to make both exceptions look similar.

Reply:
You are right, these are identical errors. Fixed.